examples/glue_benchmark/glue.py

```diff
@@ -112,200 +69,127 @@ def split_features(x):
         label = x["label"]
         return (features, label)
 
-    if task_name == "ax":
-        # AX is trained and evaluated on MNLI, and has its own test split.
-        train_ds, validation_ds = tfds.load(
-            "glue/mnli",
-            split=["train", "validation_matched"],
-        )
-        test_ds = tfds.load(
-            "glue/ax",
-            split="test",
-        )
-    else:
-        train_ds, test_ds, validation_ds = tfds.load(
-            f"glue/{task_name}",
-            split=["train", "test" + test_suffix, "validation" + test_suffix],
-        )
-
-    # Extract out the index order of test dataset.
-    idx_order = test_ds.map(lambda data: data["idx"])
+    train_ds, test_ds, validation_ds = tfds.load(
+        "glue/mrpc",
+        split=["train", "test", "validation"],
+    )
 
     train_ds = train_ds.map(split_features, num_parallel_calls=tf.data.AUTOTUNE)
     test_ds = test_ds.map(split_features, num_parallel_calls=tf.data.AUTOTUNE)
     validation_ds = validation_ds.map(
         split_features, num_parallel_calls=tf.data.AUTOTUNE
     )
-    return train_ds, test_ds, validation_ds, idx_order
+    return train_ds, test_ds, validation_ds
+
+
+def load_model_and_preprocessor(model, preset, num_classes):
+    for name, symbol in keras_nlp.models.__dict__.items():
+        if inspect.isclass(symbol) and issubclass(symbol, keras.Model):
+            if model and name != model:
+                continue
+            if not hasattr(symbol, "from_preset"):
+                continue
+            for _preset in symbol.presets:
+                if preset and _preset != preset:
+                    continue
+                if "Backbone" in name:
```

Comment:
i think we can use the same method as [here](https://github.com/keras-team/keras-nlp/blob/23e0508feb5002c02e9c91dce2742feb1460ae6b/keras_nlp/benchmarks/sentiment_analysis.py#L81) to simplify the code. Later on we can move this method to `utils/` since we are reusing it.

Reading this function, I think you are trying to enable users to pass `--model="Bert"`, which is a nice idea actually! But let's for now still force users to pass "BertClassifier"  as the flag so that we could have a standardized workflow.